blocks.md

```diff
@@ -404,6 +405,34 @@ It requires a Github [personal access token](https://github.com/settings/tokens/
 
 For more information about reasons, please see the [API documentation](https://developer.github.com/v3/activity/notifications/#notification-reasons).
 
+## Hueshift
+
+Creates a block which display the current color temperature in Kelvin. When scrolling upon the block the color temperature is changed.
+A left click on the block sets the color temperature to the mean value from the `max_temp` and `min_temp`.
```

Comment:
AFAIK the default, no-adjustment temperature is 6500k, so ideally the defaults for min and max should have 6500 as the middle value? Right now it's 5500k. Or even better, set the left click value to something configurable, instead of the middle value. Because as it is right now, since 10000 is the current maximum, it forces the user to set the minimum to 3000 (which is subjectively pretty high for a minimum) in order for the middle value to be 6500.

Maybe I'm just misunderstanding the intended use case you chose the middle value for left click though

Reply:
I think you're right it's better to set it configurable for the left click.

The reason I choose the middle value was because I wanted to always have a `current_temp` between the range of `min_temp` and `max_temp`, but by default I think the middle value should be 6500K, it's much more natural.

In case, the `max_temp` is lower than 6500, then I'll just (by default) set the left click value to the `max_temp` in that case, I'm not too sure it's a good idea, but basically the key idea here is that we're trying to get as close to the default value as we can.

Concerning the bug, I've been having somewhat of a bug too, when I had the `backlight` block and this block in the same status bar, when scrolling on the backlight, the hueshift block changes color temperature as well, I'm not too sure how I would go around limitting the interaction to only the mouse hovering over the specific block text. I'll have a look at already implemented blocks to try and fix it.

Thank you for the useful feedback!